backend/main/shared/structs.go

```diff
@@ -65,6 +67,31 @@ type MintParams struct {
 	RoyaltyBeneficiaries []string
 }
 
+type FTBalanceResponse struct {
+	ID                      string    `json:"id,omitempty"`
+	FungibleTokenID         string    `json:"fungibleTokenId"`
+	Addr                    string    `json:"addr"`
+	PrimaryAccountBalance   uint64    `json:"primaryAccountBalance"`
+	SecondaryAddress        string    `json:"secondaryAddress"`
+	SecondaryAccountBalance uint64    `json:"secondaryAccountBalance"`
+	Balance                 uint64    `json:"balance"`
+	StakingBalance          uint64    `json:"stakingBalance"`
+	ScriptResult            string    `json:"scriptResult"`
+	Stakes                  []string  `json:"stakes"`
+	BlockHeight             uint64    `json:"blockHeight"`
+	Proposal_id             int       `json:"proposal_id"`
+	NFTCount                int       `json:"nftCount"`
+	CreatedAt               time.Time `json:"createdAt"`
+}
```

Comment:
this now has fields for both FT and Flow snapshot responses, so the same struct can be used for both cases.